src/main/java/com/iexec/core/task/TaskService.java

```diff
@@ -318,17 +322,32 @@ public void consensusReached2Reopened(Task task) {
         Optional<ChainReceipt> optionalChainReceipt = iexecHubService.reOpen(task.getChainTaskId());
 
         if (!optionalChainReceipt.isPresent()) {
-            log.error("Reopen failed [chainTaskId:{}, canReopen:{}, hasEnoughGas:{}]",
-                    task.getChainTaskId(), canReopen, hasEnoughGas);
+            log.error("Reopen failed [chainTaskId:{}]", task.getChainTaskId());
             updateTaskStatusAndSave(task, REOPEN_FAILED);
             updateTaskStatusAndSave(task, FAILED);
             return;
         }
+        reopening2Reopened(task, optionalChainReceipt.get());
+    }
+
+    public void reopening2Reopened(Task task) {
+        reopening2Reopened(task, null);
+    }
 
-        task.setConsensus(null);
-        task.setRevealDeadline(new Date(0));
-        updateTaskStatusAndSave(task, REOPENED, optionalChainReceipt.get());
-        updateTaskStatusAndSave(task, INITIALIZED, optionalChainReceipt.get());
+    public void reopening2Reopened(Task task, ChainReceipt chainReceipt) {
```

Comment:
I dont think we need `public void reopening2Reopened(Task task, ChainReceipt chainReceipt)` anymore

Reply:
It is now true for few of those methods, I'll refactor that part in a future PR.